test/gpdb/test_in_docker.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+function install_pgbackrest() {
+    pushd /home/gpadmin/pgbackrest/src
+    ./configure --enable-test
+    make -j`nproc` -s
+    make install
+    make clean
+    popd
+}
+
+function stop_and_clear_gpdb() {
+    source gpdb_src/gpAux/gpdemo/gpdemo-env.sh
+    DATADIR="${MASTER_DATA_DIRECTORY%*/*/*}"
+    # Attempt to stop the Greenplum Database cluster gracefully
+    echo "Attempting to stop GPDB cluster gracefully..."
+    if ! su - gpadmin -c "
+        export MASTER_DATA_DIRECTORY=$MASTER_DATA_DIRECTORY
+        source /usr/local/greenplum-db-devel/greenplum_path.sh &&
+        gpstop -af "; then
+        # In case gpstop didn't work, force stop all postgres processes
+        # This approach is a catch-all attempt in case the cluster is in a
+        # very broken state
+        echo "Graceful stop failed. Attempting to force stop any \
+        remaining postgres processes..."
+        pkill -9 postgres
+    fi
+    sleep 2 # Giving it a moment before proceeding to the next steps
+
+    # Now that all processes are ensured to be stopped,
+    # clear the data directories.
+    echo "Clearing GPDB data directories..."
+    rm -rf $DATADIR/*
+}
+
+
+
+function run_tests() {
```

Comment:
maybe use one empty line between functions like between `install_pgbackrest` and `stop_and_clear_gpdb`?
```suggestion
}

function run_tests() {
```